src/main/java/HttpServer/ResponseWriter.java

```diff
@@ -1,19 +1,16 @@
 package HttpServer;
 
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.HashMap;
 
 public class ResponseWriter {
 
     private RequestParser parser;
     private Writable writing;
-    private ArrayList<String> resources;
+    private Controller routes;
 
     public ResponseWriter(RequestParser parser, Writable writing) {
```

Comment:
This is essentially duplicate feedback from an earlier pull request, but it's a bit odd that the `ResponseWriter` takes a `RequestParser`. I would expect it to take either A) just a `Request` or B) a `Request` and something to write to.

Reply:
Fixed.